classes/controller/ProductListingFrontController.php

```diff
@@ -308,6 +312,11 @@ protected function getProductSearchVariables()
             $query
         );
 
+        Hook::exec('actionProductSearchProviderRunQueryAfter', [
+            'query'  => $query,
```

Comment:
```suggestion
            'query' => $query,
```

Reply:
My version is easier to read, because rows are aligned to columns. May you tell me why you think your variant of alignment is better? As far as I know, PrestaShop has no an agreement on how to align lines.